extensions/wikia/CommunityPage/styles/components/_modal.scss

```diff
@@ -19,28 +19,28 @@
 		}
 
 		a {
-			color: $color-nav-inactive-text;
+			color: $color-alternate-text;
```

Comment:
According to designs provided active tab that has link color and other tabs are grayed.